CMakeLists.txt

```diff
@@ -21,6 +21,16 @@ configure_file(mender-version.h.in mender-version.h)
 
 project(mender)
 
+# The CMake documentation is quite ambiguous on the subject of available build types, and I haven't
+# been able to find an authoritative list of the default available ones. However, the manual lists
+# Debug, Release, RelWithDebInfo and MinSizeRel as "typical" types, so let's use those.
+#
+# Source:
+# https://cmake.org/cmake/help/v3.27/variable/CMAKE_BUILD_TYPE.html#variable:CMAKE_BUILD_TYPE
+if("${CMAKE_BUILD_TYPE}" STREQUAL "")
+  set(CMAKE_BUILD_TYPE "Release")
```

Comment:
I was actually thinking we would set the default to `MinSizeRel` ?

Reply:
Yeah, maybe that's a better default. I suppose more people will care about size than about speed. I/O and network are nearly always the bottlenecks anyway.